### PR TITLE
Add a few important logs

### PR DESCRIPTION
**Pull request by pmnoxx**
In this change I want to add logging http server addr, I've seen a cases, where a node would not start, because addr was already in use. 

I added a `DelayDetector` to print the time it takes to read config.

In near-performance-metrics I also changed a few `warn!` to be `info!`

### CODE REVIEW

**Comment by pmnoxx** [comment]
This PR was extracted from https://github.com/near/nearcore/pull/4012/files

File: chain/jsonrpc/Cargo.toml [hunk 1]
```diff
@@ -17,6 +17,7 @@ serde_json = "1"
 validator = "0.12"
 tracing = "0.1.13"
 borsh = "0.8.1"
+log = "0.4"
```
Comment (matklad) [inline chain/jsonrpc/Cargo.toml:20]: Why do we need both `tracing` and `log`? `tracing` is a struct super-set

Comment (matklad) [reply chain/jsonrpc/Cargo.toml:20]: https://docs.rs/tracing/0.1.25/tracing/#for-log-users

**Review by bowenwang1996 — COMMENTED** [review]
I think it is better to add `target` in the logging macro

**Comment by pmnoxx** [comment]
> I think it is better to add `target` in the logging macro

done

**Pull request merged**